pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -1771,19 +1786,6 @@ func handleDelete(ctx *zedrouterContext, key string,
 
 	appNumFree(ctx, status.UUIDandVersion.UUID)
 	appNumsOnUNetFree(ctx, status)
```

Comment:
You need to explain to me the difference between the abstraction provided by ipaddronunet.go and appnumsonunet.go.
Presumably the former is layered on top of the later, in which case zedrouter.go should only call the former abstraction. 

With your changes it is extremely hard to see where the freeing of various resources happen since there is no consistent abstraction. And if there is no such abstraction the easiest to read is the have the indivual calls at the top layer and not introduce intermediate functions; the functions should follow an intentional abstraction as supposed to just being collections of code.